catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/util/EndpointUtilTest.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyList;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.opengis.filter.Filter;
+import org.opengis.filter.Or;
+
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.AttributeRegistry;
+import ddf.catalog.data.InjectableAttribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.filter.AttributeBuilder;
+import ddf.catalog.filter.ContextualExpressionBuilder;
+import ddf.catalog.filter.EqualityExpressionBuilder;
+import ddf.catalog.filter.FilterBuilder;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.operation.impl.QueryRequestImpl;
+
+public class EndpointUtilTest {
+
+    EndpointUtil endpointUtil;
+
+    List<MetacardType> metacardTypeList;
+
+    MetacardType metacardTypeMock;
+
+    CatalogFramework catalogFrameworkMock;
+
+    FilterBuilder filterBuilderMock;
+
+    List<InjectableAttribute> injectableAttributeList;
+
+    InjectableAttribute injectableAttributeMock;
+
+    AttributeRegistry attributeRegistryMock;
+
+    Filter filterMock;
+
+    AttributeBuilder attributeBuilderMock;
+
+    ContextualExpressionBuilder contextualExpressionBuilderMock;
+
+    QueryResponse responseMock;
+
+    Metacard metacardMock;
+
+    Result resultMock;
```

Comment:
✏️  Do all of these variables need to be class-wide? It looks like `metacardTypeList`, `metacardTypeMock`, `catalogFrameworkMock`, `injectableAttributeList`, `injectableAttributeMock`, `attributeRegistryMock`, `filterMock`, `attributeBuilderMock`, and `contextualExpressionBuilderMock` are only ever used in the `setup()` method.